src/app/project/add.tpl.html

```diff
@@ -39,6 +39,7 @@
                        ng-model="vm.project_name"
                        ng-model-options="{ debounce: 500 }"
                        project-name-available-validator
+                       organization-id="vm.currentOrganization.id"
```

Comment:
Will this error out if currentOrganization is null or undefined? I think we are setting it to `{}` and we should be fine.